### DISCUSSION TITLE
landcover change

### DISCUSSION THREAD

**Discussion by vibinj** [NASA-LIS/LISF#978] [Q&A] [answered]
I have replaced the CROP land in one grid point by Evergreeen needle leaf forest and the related SURFACETYPE, LANDCOVER., GREENNESS, TEXTURE, ALBEDO, SHDMIN and SHDMAX variables also changed. Is it necessary to change variables in the VEGETATION PARAMETER.TBL, if we change the landcover class of a grid point by some another class (CROP to EVERGREEN NEEDLE LEAF FOREST) or it wiil automatically read the corresponding values from the VEGETATION PARAMETER.TBL. We are using MODIS IGBP data set.

**Comment by karsenau** [comment, marked as answer]
Hello @vibinj:

If you change the landcover value for a grid point to another existing landcover class within the same classification, then you should have no problem using the same vegetation parameter table for your runs.  You have simply just changed the value of the land class but you haven't changed the land classification scheme, so you should be fine.

**Reply by vibinj** [reply]
Dear Karsenau,
 Thanks. I have changed the land cover class within the same classification only.

**Comment by karsenau** [comment]
You're welcome, and please let us know if you have any questions or if you feel your question was answered. Thanks!
